projects/baby_driver/scripts/gpio_set.py

```diff
@@ -0,0 +1,32 @@
+"""This module provides the Python implementation to set GPIO pins using the CAN protocol"""
+
+import can_util
+
+def gpio_set(port, pin, state):
+    """
+    Sets a GPIO pin
+    Args:
+        port: port of the GPIO pin to set
```

Comment:
Request: could you make it so that you can pass in a string like `'a'` or `'A'` as the port and have that be treated like the appropriate port number? Look at Selina's adc_read PR for how she did it.